person_class.rb

```diff
@@ -1 +1,18 @@
+class Person
+  attr_accessor :name, :age, :parent_permission
+  attr_reader :id
 
+  def initialize(age, parent_permission: true, name: 'Unknown')
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def of_age?
+    @age >= 18
+  end
```

Comment:
- [ ] `of_age` method is private as mentioned in the project requirements, kindly fix that 😁 
- [ ] But don't forget if you add this method private remember the `can_use_services` is not a private method but a public method, you can add `private` before `of_age` and then add `public` before `can_use_services` to make it public or you can move the `of_age` method and it can come after the `can_use_services` method 🙃 

```ruby 
  private 
  def of_age?
    @age >= 18
  end

 public
 def can_use_services? 
    of_age? || @parent_permission
  end
```

or

```ruby 
 def can_use_services? 
    of_age?() || @parent_permission
  end
  private 
  def of_age?
    @age >= 18
  end
```